tests/algorithms/test_gradient_clipping.py

```diff
@@ -86,6 +86,7 @@ def test_gradient_clipping_algorithm(monkeypatch, clipping_type, simple_model_wi
     state.profiler.marker = Mock(return_value=None)
```

Comment:
Would it be possible to use a real state object instead of a mock here? Mainly concerned about this test still passing if we change the state object.

Reply:
fixed here: [b95594b](https://github.com/mosaicml/composer/pull/1166/commits/b95594b13447fcfbfb20b6400456429e19e9493a)